CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 - Fixed `case var` handling [#58](https://github.com/HaxeCheckstyle/tokentree/issues/58)
 - Fixed `case Pattern(var foo, var bar)` handling [#59](https://github.com/HaxeCheckstyle/tokentree/issues/59)
 - Fixed comments in if…else [#60](https://github.com/HaxeCheckstyle/tokentree/issues/60)
+- Fixed modifiers for `final` were missing [#61](https://github.com/HaxeCheckstyle/tokentree/issues/61)
```

Comment:
Not sure if this needs to be mentioned in the changelog? Before this version, `final` wasn't handled at all, so I don't think this issue existed either. :)